userinfo-api-faq.md

```diff
@@ -26,6 +26,26 @@ For more common Vipps questions, see:
 
 <!-- END_COMMENT -->
 
+## What is the `sub`?
+
+The `sub` is a unique user identifier for a Vipps user, related to that user's consent
+to share information with a specific MSN (sales unit).
+
+The `sub` is _based on_ the user's national identity number ("fødselsnummer" in Norway),
+but  is not a replacement for NIN (National Identity Number) or any other unique identifier
+for the user. 
+
+The `sub` is unique for each MSN (sales unit).
+User may have many different `sub`s for the same merchant: One for each of
+the merchant's MSNs (sales units).
+You can not use the `sub` for one MSN with the API keys for a different MSN.
+
+The `sub` will not change if a user removes their consents, and logs in again and re-consents.
+
+**Please note:** There are some special cases where the `sub` will change for a user:
+- If a user deletes the Vipps Profile and creates a new one.
+- If a user changes the phone number (in practice: Creates a new Vipps user)
+
 ## If a user removes their consents, is the `sub` still the same?
```

Comment:
We can remove line 49 to lin 61, since this information is now included in "What is the `sub`?"